android/app/src/main/java/org/fossasia/openevent/fragments/AboutFragment.java

```diff
@@ -176,5 +218,92 @@ private void setUpCustomTab(String url) {
+                            Sessions.add(session);
+                        }
+                    } catch (ParseException e) {
+                        e.printStackTrace();
+                    }
+                    final List<Session> filteredModelList = filter(Sessions, searchText.toLowerCase(Locale.getDefault()));
+                    sessionsListAdapter.animateTo(filteredModelList);
+                } else {
+                    compositeDisposable.add(dbSingleton.getBookmarkIdsObservable()
+                            .subscribe(new Consumer<ArrayList<Integer>>() {
+                                @Override
+                                public void accept(@NonNull ArrayList<Integer> ids) throws Exception {
+                                    bookmarkedIds = ids;
+                                    sessionsListAdapter.clear();
+                                    for (int i = 0; i < bookmarkedIds.size(); i++) {
+                                        Integer id = bookmarkedIds.get(i);
+                                        final int index = i;
+                                        dbSingleton.getSessionByIdObservable(id)
+                                                .subscribe(new Consumer<Session>() {
+                                                    @Override
+                                                    public void accept(@NonNull Session session) throws Exception {
+                                                        mSessions.add(session);
+                                                        sessionsListAdapter.notifyItemInserted(index);
+                                                        if (index == bookmarkedIds.size())
+                                                            handleVisibility();
+                                                    }
+                                                });
+                                    }
+                                }
+                            }));
+                }
+            } catch (ParseException e) {
+                Timber.e("Parsing Error Occurred at BookmarksFragment::onResume.");
+            }
+        }
+    }
+
+    private List<Session> filter(List<Session> sessions, String query) {
+        String lowerCaseQuery = query.toLowerCase(Locale.getDefault());
+        final List<Session> filteredTracksList = new ArrayList<>();
+        for (Session session : sessions) {
+            final String text = session.getTitle().toLowerCase(Locale.getDefault());
+            if (text.contains(lowerCaseQuery)) {
+                filteredTracksList.add(session);
+            }
+        }
+        return filteredTracksList;
+    }
+
+    private void handleVisibility() {
+        if (!bookmarkedIds.isEmpty()) {
+            bookmarksRecyclerView.setVisibility(View.VISIBLE);
+        } else {
+            bookmarksRecyclerView.setVisibility(View.GONE);
+        }
+    }
+
+    @Override
```

Comment:
Avoid unneeded lifecycle methods

Here you need to dispose CompositeDisposable, close any changelistener and do any other clean up